scripts/Source/sslConfigMenu.psc

```diff
@@ -666,7 +695,11 @@ event OnSliderAcceptST(float value)
 		endIf
 		
 		SetSliderOptionValueST(value as int, "{0}%")
-		ForcePageReset()
+		
+		string StripEditorST = "StripEditor_"+Options[1]+"_"+Options[2]
+		SetTextOptionValueST(GetStripState(ItemRef), false, StripEditorST)
```

Comment:
refresh the value just for the linked option avoiding the use of the ForcePageReset() and the amount of time wasted on the full inventory.